src/partykit/gateway.ts

```diff
@@ -124,6 +124,9 @@ export class PartyKitGateway implements bs.Gateway {
     await this.ready();
     return exception2Result(async () => {
       const { store } = getStore(uri, this.sthis, (...args) => args.join("/"));
+      if (store === "meta") {
+        bs.addCryptoKeyToGatewayMetaPayload(uri, this.sthis, body);
```

Comment:
this is all it takes to add meta endpoint crypto handling. next up is adding to more connectors

Reply:
@mabels this seems like it could be wrapped like the fragment gateway